README.md

```diff
@@ -28,88 +30,115 @@ run:
 ```
 make run
 ```
+This is mostly useful for local development.
 
-At this point the operator would pick up the CRD, create a scan for the
-suite, the scan would create a pod for every node in the cluster (this will
-change, see below), execute `openscap-scan` on every node and eventually
-report the results. The scan using one rule takes about a minute.
+## Running a scan
 
-You can watch the node progress with:
+The API resource that you'll be interacting with is called `ComplianceSuite`.
+It is a collection of `ComplianceScan` objects, each of which describes
+a scan. In addition, the `ComplianceSuite` also contains an overview of the
+remediations found and the statuses of the scans. Typically, you'll want
+to map a scan to a `MachinePool`, mostly because the remediations for any
+issues that would be found contain `MachineConfig` objects that must be
+applied to a pool.
+
+To run the scans, copy and edit the example file at
+`deploy/crds/complianceoperator.compliance.openshift.io_v1alpha1_compliancesuite_cr.yaml`
+and create the Kubernetes object:
+```
+# edit the Suite definition to your liking. You can also copy the file and edit the copy.
+$ vim deploy/crds/complianceoperator.compliance.openshift.io_v1alpha1_compliancesuite_cr.yaml
+# make sure to switch to the openshift-compliance namespace
+$ oc project openshift-compliance
+$ oc create -f deploy/crds/complianceoperator.compliance.openshift.io_v1alpha1_compliancesuite_cr.yaml
+```
+
+At this point the operator would pick up the ComplianceSuite custom resource,
+create the scans for the suite, the scan would create a pod for every node
```

Comment:
I might suggest different wording here.. "creates scan pods that run on each node in the cluster"